### ISSUE TITLE
Testing remaining routes multiple times

### ISSUE THREAD

**Issue by dhoelzgen** [9elements/aygabtu#5]
Due to the internal semantics of `remaining`, it is not possible to test remaining routes multiple times, i.e. in a desktop and mobile version.

```
remaining do
  visit
  visit_with(mobile_version: true)
end
```

When visiting the second (mobile) version, the route was already removed from `aygabtu_matching_routes` and thus no matching routes is found at runtime.

**Comment by schnittchen** [comment]
Hey @dhoelzgen, can you check against #7 please?

**Comment by schnittchen** [comment]
Thanks! Closed by #7.
